### PR TITLE
Mes 6251 lift and shift providers

### PR DESCRIPTION
**Pull request by piercemorris**
## Description

Lift and shifted several providers from DES 3. Several linting issues resolved and errors identified and marked with comments where necessary.

Not all providers have been moved and another story has been created to show for the work needing to be completed.

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -13,15 +13,16 @@
...
     "no-unused-expressions": "error",
     "class-methods-use-this": "off",
     "consistent-return": "off",
     "arrow-body-style": "off",
     "padded-blocks": "off",
+    "array-callback-return": "off",
     "no-console": "warn",
     "import/prefer-default-export": "off",
     "quotes": ["error", "single"],
-    "indent": ["error", 2],
```
Comment (szabikr) [inline .eslintrc:24]: was there no quick fix for standardising the indendations?

File: src/app/providers/analytics/analytics.actions.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable max-classes-per-file */
```
Comment (szabikr) [inline src/app/providers/analytics/analytics.actions.ts]: actions can be changed to new NGRX syntax

**Review by szabikr — APPROVED** [review]
2 comments to think about

All this will be verified when the providers need to be integrated

File: .eslintrc (hunk 1 above)
Comment (piercemorris) [reply .eslintrc:24]: We are using typescript indenting rule not the eslint indent rule so they are not conflicting

**Review by tomcrawleyy — APPROVED** [review]

**Pull request merged**